### PR TITLE
View assignee time allocated

### PR DESCRIPTION
**Pull request by teneightfive**
This change adds the time an assignee has allocated to an order to the work order view.

👉 [**DEMO**](https://datahub-beta2-pr-475.herokuapp.com/omis/ebad867d-2527-4439-b5fb-d39fb7095392/work-order) 👈

## What it looks like

localhost_3001_omis_5325ad09-0dd4-480e-94be-9a4a3cde0eb9_work-order 1

### CODE REVIEW

File: config/nunjucks/filters.js [hunk 1]
```diff
@@ -110,6 +111,10 @@ const filters = {
     return dateFns.format(parsedDate, format)
   },
 
+  humaniseMinutes: (value, format = longDateFormat) => {
```
Comment (ztolley) [inline config/nunjucks/filters.js]: Humanise and Humanize. maybe we use the US spelling to be consistent.

Also doesn't moment already contain this and the functions in humanize-duration ?

Comment (teneightfive) [inline config/nunjucks/filters.js]: You're right. For some reason I missed it when I was looking for ways to support this. Will update.

File: config/nunjucks/filters.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 const nunjucks = require('nunjucks')
-const humanizeDuration = require('humanize-duration')
+const moment = require('moment')
+require('moment-duration-format')
+// const humanizeDuration = require('humanize-duration')
```
Comment (ztolley) [inline config/nunjucks/filters.js]: why comment it and not just delete?

Comment (teneightfive) [inline config/nunjucks/filters.js]: My bad.

**Pull request merged**